### PR TITLE
Disable automatic on-touch activation of external airlocks

### PR DESCRIPTION
**Pull request by 20kdc**
Why:

+ It's a total pain to reliably get inside a two-door airlock without tripping the firelocks, and that's if you're *intending* to use it properly...
+ People keep draining the entryway air on the sandbox servers (so people have to crowbar the firelocks open, and this reduces available air)

How this deals with it:

External airlocks - specifically - no longer activate on-touch.
Clicking becomes the default method for activating these.
A property `autoOpen` is added to them for this purpose, and the description has been modified to try and make this clearer.

### CODE REVIEW

File: Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs [hunk 1]
```diff
@@ -112,6 +116,7 @@ public override void ExposeData(ObjectSerializer serializer)
             base.ExposeData(serializer);
 
             serializer.DataField(ref _occludes, "occludes", true);
+            serializer.DataField(ref _autoOpen, "auto_open", true);
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs:119]: Should be autoOpen

**Comment by metalgearsloth** [comment]
IIRC the end goal was to have the airlocks cycle like they do in ss13 but it needed signal linking? With that being said I'm not averse to the change.

**Comment by 20kdc** [comment]
Cycling would probably be better, but if the signals were being done via the machine linking subsystem, that sounds like it'd require signal linking to allow access to multiple different sub-signals.

File: Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs (hunk 1 above)
Comment (20kdc) [reply Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs]: Fixed (see latest commit)

**Comment by RemieRichards** [comment]
I'd prefer `bumpOpen` instead of `autoOpen`

("auto" makes me think of powered doors and such, which isn't *all doors* like the `ServerDoorComponent` is)

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**